core/.eslintignore

```diff
@@ -1,2 +1,5 @@
 src/components/slides/swiper/swiper.bundle.js
 src/components.d.ts
+**/test/**/*.spec.ts
+**/test/**/*.spec.tsx
```

Comment:
Why do we want to ignore the spec tests?

Reply:
I mentioned that in the PR description. Basically we need to ignore the same files being excluded in the tsconfig.